data/856/326/49/85632649.geojson

```diff
@@ -287,6 +290,7 @@
     "src:lbl_centroid":"mapshaper",
     "src:population":"wd",
     "src:population_date":"2019",
+    "src:population_year":"2019",
```

Comment:
Just catching this.. but I think this should be `src:population_date`, no? I don't see a `src:population_year` property https://github.com/whosonfirst/whosonfirst-properties/tree/main/properties/src

Reply:
Great catch, thank you! I'll fix... Affects 340 records.

Reply:
Resolved via https://github.com/whosonfirst-data/whosonfirst-data-admin-us/pull/152/commits/1fb9a46222b48de5f455655cfda3399a9afae6c8 in this repo, and other commits in other repos...